tests/CMakeLists.txt

```
@@ -118,14 +118,11 @@ foreach(_category ${_categories})
 
   file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${_category})
 
+  set(_redirect > /dev/null)
```

Is this special syntax, or do you mean this?
```suggestion
  set(_redirect "> /dev/null")
```

---

This will not work as CMake will quote the argument before passing it to the command line. I could write `">" "/dev/null"` but this looks a bit funny.

---

That's still my preference. The previous syntax made it difficult to discern if you're just setting a variable, or if cmake interprets `>` in some special way. (This being cmake wouldn't make that impossible to conceive...)